vumi/middleware/tests/test_base.py

```diff
@@ -280,9 +288,12 @@ def test_parse_yaml(self):
 
     @inlineCallbacks
     def test_sort_by_priority(self):
-        priority2 = ToyMiddleware('priority2', {"priority": 2}, self)
-        priority1_1 = ToyMiddleware('priority1_1', {"priority": 1}, self)
-        priority1_2 = ToyMiddleware('priority1_2', {"priority": 1}, self)
+        priority2 = ToyMiddleware('priority2', {}, self)
```

Comment:
The second parameter is the config dict. We used to just tag the parameter on as an extra key in the config dict, but we can't do this when using a config class (unless maybe we add it to the BaseMiddlewareConfig?).

At the moment I've made it a property of the middleware class (`mw.consume_priority`). I'm not sure if this is a better way of doing it, or if it should rather be added to the BaseMiddlwareConfig.